.github/workflows/changelog.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - run: github_changelog_generator -u w3c -p respec --no-unreleased
         env:
           CHANGELOG_GITHUB_TOKEN: ${{ secrets.CHANGELOG_GITHUB_TOKEN }}
```

Comment:
The docs say [`secrets.GITHUB_TOKEN` is autogenerated inside an Action](https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#about-the-github_token-secret), is there a reason not to use it?

Reply:
The default has 1000 API request limit per hour (it used to be 10k), so we run out of quota. Using my personal token here.

(This changelog generator needs to be rewritten by someone)